### PR TITLE
[wip] fixed almalinux issue

### PR DESCRIPTION
**Pull request by BalramY**
https://github.com/AlmaLinux/almalinux.org/issues/34 //done

Then #35 https://github.com/AlmaLinux/almalinux.org/issues/35  //done
Then #36 https://github.com/AlmaLinux/almalinux.org/issues/36  //done
Then #37 https://github.com/AlmaLinux/almalinux.org/issues/37 //done

### CODE REVIEW

File: www/templates/layouts/base.html [hunk 1]
```diff
@@ -56,6 +73,49 @@
         <meta name="twitter:card" content="summary_large_image">
         <meta property="og:locale" content="{{ CURRENT_LANG }}" />
     {% endblock %}
+
+    <script>
+        function MemberShipInfo(id) {
```
Comment (Addvilz) [inline www/templates/layouts/base.html:78]: No inline JavaScript please.

File: www/templates/layouts/base.html [hunk 2]
```diff
@@ -45,6 +46,22 @@
         a {
             color: #c4e1ff;
         }
+        .new-law-btn{
```
Comment (Addvilz) [inline www/templates/layouts/base.html:49]: No inline CSS please. CSS above is to fix UX issue.

File: www/templates/layouts/base.html [hunk 3]
```diff
@@ -35,6 +35,7 @@
     <meta name="theme-color" content="#0f4266">
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&display=swap" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
```
Comment (Addvilz) [inline www/templates/layouts/base.html:38]: Use of jQuery is detrimental to the performance of the site and we don't want it here.

File: www/templates/foundation/members/index.html [hunk 4]
```diff
@@ -50,9 +50,13 @@ <h1 class="display-5 fw-bold mb-4">
                         <p>
                             {% blocktranslate trimmed %}
                             The merit of each application will be reviewed on a case-by-case basis by the Board or by a
-                            committee appointed for this purpose, according to the Bylaws established by the Board.
+                            committee appointed for this purpose, according to the 
+                            <a href="https://almalinux.org/p/foundation-bylaws/" style="text-decoration: none;"><b>Bylaws</b></a> established by the Board.
```
Comment (Addvilz) [inline www/templates/foundation/members/index.html:54]: No inline styling, please.

File: www/templates/foundation/members/index.html [hunk 5]
```diff
@@ -50,9 +50,13 @@ <h1 class="display-5 fw-bold mb-4">
...
                             {% endblocktranslate %}
                         </p>
+                        <p>
+                            <a href="https://drive.google.com/file/d/1xYkib2FECMoVy5olG9R9audYfGXpOLjf/view?usp=sharing" style="text-decoration: none;"><b>Membership guidelines are available here</b></a>
```
Comment (Addvilz) [inline www/templates/foundation/members/index.html:58]: No inline styling, please.

File: www/templates/contribute/index.html [hunk 6]
```diff
@@ -82,6 +82,13 @@ <h3 class="fw-bold pb-3">
                             {% translate 'Join AlmaLinux OS community chat' %}
                         </a>
                     </div>
+                    <div>
+                        <a href="https://almalinux.org/foundation/members/"
+                           class="btn al-cta-chat btn-lg px-4 mt-3">
```
Comment (Addvilz) [inline www/templates/contribute/index.html:87]: `.al-cta-chat` is not meant to be used in this template.

File: www/templates/page/page.html [hunk 7]
```diff
@@ -19,7 +19,7 @@
 
     <section class="al-page-page" dir="{% text_direction page.language %}">
         <div class="container py-5">
-            <h1 class="al-section-title mb-4">{{ page.title }}</h1>
+        <!-- <h1 class="al-section-title mb-4">{{ page.title }}</h1>-->
```
Comment (Addvilz) [inline www/templates/page/page.html:22]: Why this change?

**Review by Addvilz — CHANGES_REQUESTED** [review]
This PR does need some significant work done.

- We do not want to introduce jQuery to the project.
- Please avoid inline JavaScript and CSS. All of the styling and FE logic should be handled in frontend components.
- Avoid use of tables for non-tabular data.

**Comment by BalramY** [comment]
> This PR does need some significant work done.
> 
> * We do not want to introduce jQuery to the project.
> * Please avoid inline JavaScript and CSS. All of the styling and FE logic should be handled in frontend components.
> * Avoid use of tables for non-tabular data.

Now Fixed

File: frontend/src/modules/page_foundation/main.js [hunk 8]
```diff
@@ -1 +1,46 @@
 import './main.scss';
+document.getElementById("contributor").addEventListener('click',function ()
```
Comment (Addvilz) [inline frontend/src/modules/page_foundation/main.js:2]: - This will fail to correctly initialize in some cases. Please, wrap this code in DOMContentLoaded event handler.
- Please use quotes consistent with the rest of the codebase - e.g. single quotes.
- Please terminate all lines with semicolon `;`
- Replace `var` with `const` where appropriate.

File: www/templates/foundation/members/index.html [hunk 9]
```diff
@@ -50,9 +50,17 @@ <h1 class="display-5 fw-bold mb-4">
                         <p>
```
Comment (Addvilz) [inline www/templates/foundation/members/index.html:50]: All of the text that was added in this page must be marked with translate or blocktranslate.

File: www/templates/foundation/members/index.html [hunk 10]
```diff
@@ -50,9 +50,17 @@ <h1 class="display-5 fw-bold mb-4">
...
                             {% blocktranslate trimmed %}
                             The merit of each application will be reviewed on a case-by-case basis by the Board or by a
-                            committee appointed for this purpose, according to the Bylaws established by the Board.
+                            committee appointed for this purpose, according to the 
+                            <a href="https://almalinux.org/p/foundation-bylaws/" id="al-laws-link">
```
Comment (Addvilz) [inline www/templates/foundation/members/index.html:54]: - Links should not be part of blocktranslate. Please see other templates on examples how to handle links in translatable text.
- Styling for inline links should be anchored from parent element (e.g. styling the `p`) not using an inline ID.

**Review by Addvilz — CHANGES_REQUESTED** [review]

**Comment by Addvilz** [comment]
Please, roll back all the unnecessary file mode changes.

**Comment by BalramY** [comment]
I created a new PR for this https://github.com/AlmaLinux/almalinux.org/pull/39 

Please review that one. I will close this PR. No need to merge it.
